Dockerfile

```diff
@@ -9,7 +9,8 @@ FROM ubuntu:18.04 AS builder
 
 ARG DEFAULT_MODE=default
 ENV MODE=$DEFAULT_MODE
-RUN --mount=target=/polycube cp -r /polycube /tmp/polycube && cd /tmp/polycube && \
+COPY /polycube/ /tmp/polycube 
```

Comment:
Shouldn't it be `COPY . /tmp/polycube/`? I'm scared that `COPY /polycube/ ...` fails because, when building, the process is already inside the directory